app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -20,81 +21,110 @@ import DeviceContext from '../context/device';
 import UploadContext from '../context/upload';
 import useCounter from '../hooks/use-counter';
 import useCookie from '../hooks/use-cookie';
+import type { ReactNode, MouseEvent } from 'react';
```

Comment:
While I'm glad to see this converted to TypeScript, what would you think of creating a separate pull request just for that porting (since it's a large changeset), to more easily distinguish the changes for supporting the work of this ticket?

Reply:
I could certainly do that. Given the actual ticket level things we changed, I will have to update the JSDoc type annotations anyway in several places. But of course those will be smaller changes